### PR TITLE
Edmc pin work

### PR DESCRIPTION
**Pull request by skiedude**
1. react EDMC OFF in emojis to the trade alert when edmc_off is True #384  
2. fix edmc typos #386 
3. clean up the bot pinned messages on m.done|m.complete  #383 
4. erase the system notification message that a message was pinned in all public guild channels #387 

Demo: https://imgur.com/4gh8F8U

### CODE REVIEW

**Review by cimspin — CHANGES_REQUESTED** [review]
Also can we add to the Discord trade alert text (defined from line 850, `txt_create_discord`) the prefix `**EDMC-OFF MISSION: **` for EDMC-off missions? Just to thoroughly distinguish them from normal missions. Something like:

```def txt_create_discord(carrier_data, mission_type, commodity, station, system, profit, pads, demand, eta_text, mission_temp_channel_id, edmc_off):
    discord_channel = f"<#{mission_temp_channel_id}>" if mission_temp_channel_id else f"#{carrier_data.discord_channel}"
    discord_text ={" **EDMC-OFF MISSION: **" if edmc_off else ""} \
                   f"{discord_channel} {'load' if mission_type == 'load' else 'unload'}ing " \
                   f"{commodity.name} " \
                   f"{'from' if mission_type == 'load' else 'to'} **{station.upper()}** station in system " \
                   f"**{system.upper()}** : {profit}k per unit profit : "\
                   f"{demand} {'demand' if mission_type == 'load' else 'supply'} : {pads.upper()}-pads" \
                   f".{eta_text}"
    return discord_text```
    
You'd have to pass `edmc_off` to the `txt_create_discord function` from whereever `discord_text` is defined as well of course. But, eh, feel free to do it however you want, just slowly dusting off the coding section of my brain

**Comment by cimspin** [comment]
Also, amazing work as usual

**Comment by skiedude** [comment]
New trade alert message look:
edmc_off

System notifications for pinned messages for normal users
pin

New prompt with the `e` option
prompt

File: MissionAlertBot.py [hunk 1]
```diff
@@ -1219,39 +1234,24 @@ async def default_timeout_message(location):
...
-            edmc_off = False
-            if 'y' == edmc_msg.content.lower().strip():
-                edmc_off = True
-        except asyncio.TimeoutError:
-            await default_timeout_message('edmc_prompt')
-            return
-
         embed = discord.Embed(title="Where would you like to send the alert?",
-                            description="(**d**)iscord, (**r**)eddit, (**t**)ext for copy/pasting or (**x**) to cancel\n"
+                            description="(**d**)iscord, (**e**)dmc OFF, (**r**)eddit, (**t**)ext for copy/pasting or (**x**) to cancel\n"
```
Comment (cimspin) [inline MissionAlertBot.py:1238]: `description="(**d**)iscord, (**r**)eddit, (**t**)ext for copy/pasting or (**x**) to cancel\n"`

File: MissionAlertBot.py [hunk 2]
```diff
@@ -1219,39 +1234,24 @@ async def default_timeout_message(location):
...
                             "Use (**n**) to also notify PTN Haulers.",
```
Comment (cimspin) [inline MissionAlertBot.py:1239]: `"Use (**n**) to notify PTN Haulers, and (**e**) to flag your mission as EDMC-OFF.",`

**Review by cimspin — CHANGES_REQUESTED** [review]
Small nitpick on lines 1238-9 for stylistic reasons, otherwise good to go 👍

**Comment by skiedude** [comment]
edmc

**Review by cimspin — APPROVED** [review]
Awesome stuff

**Pull request merged**